py/selenium/webdriver/common/actions/pointer_input.py

```diff
@@ -15,17 +15,22 @@
 # specific language governing permissions and limitations
 # under the License.
 from .input_device import InputDevice
+from .interaction import POINTER, POINTER_KINDS, POINTER_MOUSE
 
+from selenium.common.exceptions import InvalidArgumentException
 from selenium.webdriver.remote.webelement import WebElement
 
 
 class PointerInput(InputDevice):
 
     DEFAULT_MOVE_DURATION = 250
 
-    def __init__(self, type_, name):
+    def __init__(self, name, kind=POINTER_MOUSE):
```

Comment:
Keep `kind` as the first argument and it _shouldn't_ be a breaking change. There should be no reason why people would have been using their own PointerInput since `mouse` is the only pointer currently supported. Just in case though we should wait for a point release to make the change. We do need to update the 1 use within the library (below) to use interaction.POINTER_MOUSE instead of interaction.POINTER. 

https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/common/actions/pointer_actions.py#L30

Reply:
sounds good, will make these changes as soon as I can